docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,391 +1,498 @@
+## Create visual
 
-    The following example illustrats part of the [Tizen.NUI.ColorVisual](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.ColorVisual.html) class, which defines a color visual property map:
+To create a visual using `BorderVisual` class, follow these steps:
 
+1.  Create a visual object of a desirable type:
+    ```csharp
+    BorderVisual _borderVisual = new BorderVisual();
     ```
-    public class ColorVisual : VisualMap
-
-    private Color _mixColorForColorVisual = null;
-
-    public Color Color
-    {
-        get
-        {
-            return _mixColorForColorVisual;
-        }
-        set
-        {
-            _mixColorForColorVisual = value;
-            UpdateVisual();
-        }
-    }
-    ```
-
--   You can create a visual from an output visual map:
 
+2.  Specify the properties required for the visual of that type.
+    ```csharp
+    // Set the border thickness
+    _borderVisual.BorderSize = 10.0f;
+    // The border color
+    _borderVisual.Color = Color.Blue;
     ```
-    var colorMap = new ColorVisual{Color=Color.White;};
-    var _colorVisual = VisualFactory.Instance.CreateVisual(colorMap.OutputVisualMap);
-    RegisterVisual(ColorVisualPropertyIndex, _colorVisual);
-    ```
-
-    You can also create a visual using an output map in a control property. The following example uses the `Background` property:
 
+3.  You can specify the optional properties of a given visual, and also specify the properties that are common for all visual types:
+    ```csharp
+    // The size of a visual given as a percentage of the size of the parent
+    _borderVisual.RelativeSize = new RelativeVector2(0.5f, 0.5f);
+    // The parent's reference point
+    _borderVisual.Origin = Visual.AlignType.TopBegin;
+    // The Visual's reference point
+    _borderVisual.AnchorPoint = Visual.AlignType.TopBegin;
+    // Shift between Origin and AnchorPoint given as a percentage of a parent size 
+    _borderVisual.RelativePosition = new RelativeVector2(0.25f, 0.25f);
     ```
-    ColorVisual colorVisualMap1 = new ColorVisual();
-    colorVisualMap1.Color = Color.Green;
-    _visualView.Background = colorVisualMap1.OutputVisualMap;
 
-    window.GetDefaultLayer().Add(_visualView);
+4.  To add a visual to an already existing `VisualView`, use `AddVisual()` as follows:
+    ```csharp
+    _visualView.AddVisual("name_of_the_visual", _borderVisual);
     ```
 
--   You can position and resize visuals within a control using a visual transform property map.
-
-    The [Tizen.NUI.VisualTransformPropertyType](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.VisualTransformPropertyType.html) enumeration lists the transformation properties that can be defined, such as the following:
+## Use VisualView
+
+`VisualView` is a class that inherits the [Tizen.NUI.BaseComponents.View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) class. Its properties can be used to specify the `VisualView`.
```

Comment:
@mijin85cho Just for learning purposes, where am I wrong in this?


`VisualView` is a class that inherits the [Tizen.NUI.BaseComponents.View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) class, you can use its properties to specify the `VisualView`.


https://github.com/Samsung/tizen-docs/pull/1378#discussion_r547017651

Reply:
> @mijin85cho Just for learning purposes, where am I wrong in this?
> 
> `VisualView` is a class that inherits the [Tizen.NUI.BaseComponents.View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) class, you can use its properties to specify the `VisualView`.
> 
> [#1378 (comment)](https://github.com/Samsung/tizen-docs/pull/1378#discussion_r547017651)

@nawab-samsung, as discussed kindly break it or use conjunction.